doc/user-manual/tools/interface-files.rst

```diff
@@ -25,14 +25,18 @@ results include:
 Storage
 -------
 
-In projects that do not use any Agda library, the ``.agdai`` files are
-stored alongside the ``.agda`` source file.
-
-If the ``.agda`` source file is part of a project with an *identifiable root*
-(i.e. if there is an ``.agda-lib`` file in any of the directories above it),
-then the interface file is stored in the ``_build/VERSION`` directory at the
-identified root. This prevents losing the interface file when switching between
-agda versions. You can revert this behaviour with the option :option:`--no-libraries`.
+If an Agda file has one or more
+:ref:`associated<The_agda-lib_files_associated_to_a_give_Agda_file>`
```

Comment:
"give"  -->  given
Maybe also?: "to" --> with

Looks like this is a reference to an existing section heading or so, which also needs to be fixed then.

Reply:
Thanks!